plugins/outputs/wavefront/sample.conf

```diff
@@ -50,3 +50,11 @@
   ## of metrics will block for a longer time, but this will be handled gracefully by the internal buffering in
   ## Telegraf.
   #immediate_flush = true
+
+  ## Optional TLS Config
+  #tls_ca = "/etc/telegraf/ca.pem"
+  #tls_cert = "/etc/telegraf/cert.pem"
+  #tls_key = "/etc/telegraf/key.pem"
+
+  ## HTTP Timeout
+  #timeout="3s"
```

Comment:
Where did you get 3s from? I've seen this in other plugins, but the default is actually 5s: https://github.com/influxdata/telegraf/blob/aade2e4c4150cd3b68f4dde2a2e04ad63ae93cfd/plugins/common/http/config.go#L51-L54

Reply:
Ah, that's interesting. It's an arbitrary value that I got from other config examples. I wouldn't have inferred that the example value should be the same as the default. 10s is the Wavefront default, so I'll update it to match that.